docker-compose.yaml

```diff
@@ -22,7 +22,7 @@ services:
       context: .
       dockerfile: ./docker/Dockerfile
       args:
-        - DISPLAY=${DISPLAY:-}
+              #- DISPLAY=${DISPLAY:-}
         - XAUTHORITY=~/.Xauthority
```

Comment:
Please delete XAUTHORITY

Reply:
XAUTHORITY will be needed for DISPLAY config which is used by heart disease UI client